packages/webpack-build-scripts/index.js

```diff
@@ -14,6 +14,7 @@ module.exports = {
         "@blueprintjs/labs": "var Blueprint.Labs",
         "@blueprintjs/table": "var Blueprint.Table",
         "classnames": "classNames",
+        "date-fns": "date-fns",
```

Comment:
`date-fns` is almost certainly not the name of the global variable, as hyphens do not a valid JS identifier make. maybe it's `dateFns`? can you please investigate this?

Reply:
ah you're right it's `dateFns` i'll make the change